web/public/style/main.css

```diff
@@ -68,4 +68,9 @@ main {
 #header-nav {
     grid-column-start: 3;
     text-align: right;
+}
+
+#developers {
```

Comment:
credits and other info are only visible on the home page, hidden on other pages (display: none;)